src/main/java/frc/robot/commands/climber/RunClimberCommand.java

```diff
@@ -10,42 +10,50 @@
 import frc.robot.subsystems.Climber;
 
 /**
- * Runs a {@link Climber}
+ * Runs the two {@link Climbers}
  */
 public class RunClimberCommand extends CommandBase {
-  private final Climber climber;
+  private final Climber leftClimber;
+  private final Climber rightClimber;
   private final Supplier<Double> speedSupplier;
 
   /**
-   * Runs the {@link Climber} at the given speed
+   * Runs the {@link Climbers} at the given speed
    * 
-   * @param climber The {@link Climber} to run
+   * @param leftClimber The left {@link Climber} to run
+   * @param rightClimber The right {@link Climber} to run
    * @param speed The speed to run at
    */
-  public RunClimberCommand(Climber climber, double speed) {
-    this(climber, () -> speed);
+  public RunClimberCommand(Climber leftClimber, Climber rightClimber, double speed) {
+    this(leftClimber, rightClimber, () -> speed);
   }
 
   /**
-   * Runs the {@link Climber} at the speed given by the supplier
+   * Runs the {@link Climbers} at the speed given by the supplier
```

Comment:
Replace with: `Runs the {@link Climber}s at the given speed` or something similar.

I don't think the javadoc will work if you put the s inside the {}s.

Reply:
oops, silly mistake
thanks!